script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -573,59 +677,68 @@ The sister tosses away her blood-soaked halberd with a dejected look on her face
-It was a sigh so remarkably melancholic, it could have carried away her soul.
+It was an incredibly melancholy sigh,
+ almost like her soul was leaving her body.
 }
 [sha:61f464aa7e550cc0c5351f7e9b386fc30add72d3]{
 -- Page 467, Offset 19434.
 -- 「ザコどもを纏めて処理か。
-"Gathering and dealing with the small-fry, hah?
+"You get saddled with rounding up the small-fry, eh?
 }
 [sha:2fd7b7bc87e49be4d214b12d56fb6d0176224202]{
 -- Page 467, Offset 19435.
 -- さすが下っ端、<溝|どぶ>さらいだけは慣れてるじゃねえか」
-That's a lackey for you! Well used to cleaning gutters, aren't cha?"
+That's a lackey for you!
+ Bet you're used to digging ditches, ain'tcha?"
 }
-[sha:70a544ce909034266812004d9aca986952265be1]{
+[offset:19436]{
 -- Page 467, Offset 19436.
 -- 「!?」
-"?!"
+"Huh?!"
 }
 [sha:305cad417a37b3adc9f331bb94893226d23e63f9]{
 -- Page 468, Offset 19437.
 -- 背後からの声を聞いて、シスターはあわてて槍を拾い上げた。
 // 槍 here but its probably referring to the same weapon
-Hearing a voice from behind, the sister hurriedly picks the halberd up.
+Hearing someone call out to her from behind,
+ the nun hurriedly picks her halberd back up.
 }
 [sha:96c2fe2dd4c8e3ce8724e268d8f111c089d152da]{
 -- Page 468, Offset 19438.
 -- 彼女は自分以下の敵の対処は抜群に巧いが、自分と同等以上のものにはめっぽう弱い。
-Though she excels in dealing with enemies weaker than her, she stands no chance against those of equal or greater power than her.
+Though she excels in dealing with enemies weaker than her,
+ she stands no chance against anyone as strong as, or stronger than, herself.
 }
 [sha:7c41bc3e70e08ac66e6b13d69a24105b730c4d8d]{
 -- Page 468, Offset 19439.
 -- つねに最悪のケースを想像して自分を追い詰めてしまう性格が、彼女に本来の実力を発揮させないからだ。
-Due to her always imagining the worst-case scenario and pushing her back against the wall, her personality prevents her from exhibiting her true power.
+She can't help but imagine the worst scenario that might occur,
+ and because of this personality trait she always backs herself into a corner,
+ preventing herself from achieving her true potential.
 }
 [sha:422f08de5fe9fd6a8585e02ca8d13758cc49da11]{
 -- Page 469, Offset 19440.
 -- この場合もそれに近い。
-What is happening now is akin to that.
+That's more or less exactly what's happening now.
 }
 [sha:26ad11e81448f356ba2c21ecf0ab0b9d0bdf32d6]{
 -- Page 469, Offset 19441.
 -- まだ相手を視認してさえいないのに、“自分が関与していなかった第三者が来た”という事だけで精神の均衡が崩れかけている。
-She hasn't even seen them yet, but the fact that an uninvolved outsider arrived has destroyed her mental fortitude.
+She hasn't even seen them yet,
+ but the mere thought of someone she doesn't know suddenly getting involved is enough to wreck her confidence.
 }
 [sha:264b0214b7ba4b7cc08034561c844bab9bb1836c]{
 -- Page 470, Offset 19442.
 -- 「あぁ？　あんだけの殺しをやっておいてなにブルってやがる。落ち着けよ牝豚。テメェにとっちゃ神に等しい上司さまだっつの」
-"Aaaah? Why the fuck are you cowering in your boots after killing all those fuckers? Calm down, bitch. It's me, your boss. I'm pretty much God to a fucker like you."
+// Peak mario line
+"Huh?
+ Why the hell are you shaking in your boots after you wasted all those fuckers?
```

Comment:
extra jersey points if you use "clipped" (or if you're feeling especially dangerous, "whacked") instead of "wasted"